pwndbg/glibc.py

```diff
@@ -17,18 +17,19 @@
 from elftools.elf.relocation import Relocation
 from typing_extensions import ParamSpec
 
-import pwndbg.gdblib.elf
-import pwndbg.gdblib.file
-import pwndbg.gdblib.heap
-import pwndbg.gdblib.info
-import pwndbg.gdblib.memory
-import pwndbg.gdblib.proc
-import pwndbg.gdblib.symbol
+import pwndbg.aglib.elf
+import pwndbg.aglib.file
+import pwndbg.aglib.memory
+import pwndbg.aglib.proc
 import pwndbg.lib.cache
 import pwndbg.lib.config
 import pwndbg.search
 from pwndbg.color import message
 
+if pwndbg.dbg.is_gdblib_available():
+    import pwndbg.aglib.heap
```

Comment:
is there reason why `import pwndbg.aglib.heap` is in this if?

Reply:
Oh, yeah, that's something I'd overlooked. That module was ported before `pwndbg.gdblib.heap` was ported to `aglib`, and I forgot to change it after the port. Should be good now.